main.go

```diff
@@ -353,5 +353,11 @@ func loadConfig(configPath string) (*config, error) {
 	if err = yaml.Unmarshal(contents, &cfg); err != nil {
 		return nil, err
 	}
+
+	for idx, condition := range cfg.RejectionConditions {
+		if condition.ErrorCode != "" && strings.HasPrefix(condition.ErrorCode, "RS_REJECTED_") == false {
+			return nil, fmt.Errorf("Rejected condition %d was invalid. ErrorCode must be use the namespace RS_REJECTED_", idx);
```

Comment:
it's "errorcode" from the admin's point of view, not "ErrorCode"

```suggestion
			return nil, fmt.Errorf("Rejection condition %d was invalid. `errorcode` must be use the namespace `RS_REJECTED_`; got: `%s`", idx, condition.ErrorCode);
```